lib/Runtime/Language/InlineCache.cpp

```diff
@@ -26,8 +26,6 @@ namespace Js
         // We may, however, have a flags cache (setter) change to add property cache.
         Assert(typeWithoutProperty == nullptr || !IsProto());
 
-        requestContext->RegisterAsScriptContextWithInlineCaches();
```

Comment:
This would have added the script context into the list to be zero before.
Now it doesn't set the flag to indicate the script context needed to be zeroed?